app/controllers/insured/family_members_controller.rb

```diff
@@ -41,7 +42,9 @@ def index
       special_enrollment_period.qualifying_life_event_kind = qle
       special_enrollment_period.qle_on = Date.strptime(params[:qle_date], "%m/%d/%Y")
       special_enrollment_period.qle_answer = params[:qle_reason_choice] if params[:qle_reason_choice].present?
-      special_enrollment_period.save
+      if special_enrollment_period.save
```

Comment:
currently we've two workflows for creating sep...one is through family > manage family page, second through plan shopping/landing page.

Please move this notice trigger on to model using model events, that way it would cover both the scenarios.

Reply:
sure will do